enatega-multivendor-app/src/ui/FdSocialBtn/FdGoogleBtn/FdGoogleBtn.js

```diff
@@ -17,7 +16,7 @@ const FdGoogleBtn = props => {
       onPressIn={props.onPressIn}
       onPress={props.onPress}>
       {props.loadingIcon ? (
-        <Spinner backColor="rgba(0,0,0,0.1)" spinnerColor={'#FFF'} />
+        <Spinner backColor="#fff" spinnerColor={'#90E36D'} />
```

Comment:
Try using theme colors to tackle dark and light both themes.

Reply:
done